lib/team_factory.rb

```diff
@@ -16,4 +16,9 @@ def create_teams
     end
     @teams
   end
+  
+  def count_of_teams
```

Comment:
I was including methods in the non factory class but this definitely works, I was actually wondering if we might want to abstract the methods out into a separate class altogether.